kamailio/assets/service_checks.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "agent_version": "7.0.0",
+    "integration": "kamailio",
+    "check": "kamailio.services_up",
+    "statuses": ["ok", "warning", "critical"],
+    "groups": [],
+    "name": "Kamailio Services Up",
+    "description": "Returns `CRITICAL` if the services are down, `WARNING` if the agent is misconfigured, or `OK` if the services are running."
+  },
+  {
+    "agent_version": "7.0.0",
```

Comment:
```suggestion
    "agent_version": "7.40.1",
```

Reply:
resolved in cf85a7a2cd46cc4e6c399e639ff477102a24aead